GitUI/CommandsDialogs/FormViewPatch.cs

```diff
@@ -67,9 +71,8 @@ private void LoadPatchFile()
         {
             try
             {
-                var patches = PatchManager.LoadPatchFile(
-                    PatchFileNameEdit.Text,
-                    Module.FilesEncoding);
+                var text = System.IO.File.ReadAllText(PatchFileNameEdit.Text, GitModule.LosslessEncoding);
```

Comment:
:bulb:

```csharp
using File = System.IO.File;
```

Reply:
Or rename the symbol that conflicts. `File` is not a very descriptive term for, what I assume is, a property.